meta-luneos/recipes-multimedia/pulseaudio/pulseaudio-modules-droid-hidl_git.bb

```diff
@@ -0,0 +1,33 @@
+SUMMARY = "PulseAudio Droid HIDL module"
+LICENSE = "LGPL-2.1"
+LIC_FILES_CHKSUM = "file://COPYING;md5=f294906e6e4eac9d917503a0bbd139b4"
+
+DEPENDS += "pulseaudio pulseaudio-pulsecore-private-headers libhybris virtual/android-headers dbus audiosystem-passthrough pulseaudio-modules-droid pulseaudio-module-dbus-protocol"
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+# Depends on libhybris which has this restriction
+COMPATIBLE_MACHINE = "^halium$"
+
+PULSEAUDIO_VERSION = "14.2"
```

Comment:
This should either be 15.0 or other recipe should be 14.2. I think I bumped PA to 15.0 on my local Honister build.